webapp/tests/Unit/Fixtures/sample/results-full-honors.tsv

```diff
@@ -0,0 +1,10 @@
+results	1
+10001	1	Gold Medal	4	36	20	Sample Group A
+10002	2	Gold Medal	2	5	3
+10003	2	Gold Medal	2	5	3
+10004	4	Highest Honors	1	12	12	Sample Group B
```

Comment:
It's not really defined what order this file should be in, nor what should happen when some categories have no medals. It looks a bit weird like this, but I think this is the best we can do,